mapie/regression/time_series_regression.py

```diff
@@ -143,44 +141,94 @@ def fit(
         -------
         MapieTimeSeriesRegressor
             The model itself.
+
+        Raises
+        ------
+        ValueError
+            If the length of ``y`` is greater than
+            the length of the training set.
         """
-        self = super().fit(X=X, y=y, sample_weight=sample_weight)
-        self.conformity_scores_ = self._relative_conformity_scores(
+        check_is_fitted(self, self.fit_attributes)
+        X, y = cast(NDArray, X), cast(NDArray, y)
+        m, n = len(X), len(self.conformity_scores_)
+        if m > n:
+            raise ValueError(
+                "The number of observations to update is higher than the"
+                "number of training instances."
+            )
+        new_conformity_scores_ = self._relative_conformity_scores(
             X, y, ensemble=ensemble
         )
+        self.conformity_scores_ = np.roll(
+            self.conformity_scores_, -len(new_conformity_scores_)
+        )
+        self.conformity_scores_[
+            -len(new_conformity_scores_):
+        ] = new_conformity_scores_
         return self
 
-    def partial_fit(
+    def init_alpha(
```

Comment:
shouldn't it be a hidden method ? 
```suggestion
    def _init_alpha(
```

Reply:
isn't it more like a _get_alpha than init ?

Reply:
Done